intelmq/etc/harmonization.conf

```diff
@@ -205,6 +205,10 @@
             "regex": "^[a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[0-9a-z]{12}$",
             "type": "LowercaseString"
         },
+        "misp.to_ids": {
+            "description": "MISP - Malware Information Sharing Platform & Threat Sharing IDS flag",
```

Comment:
For me, this is not descriptive enough. I can't understand the meaning of the field from this text.